src/chime_dash/app/pages/sidebar.py

```diff
@@ -160,6 +163,35 @@ def update_parameters(*input_values, **kwargs) -> List[str]:
         )
         return [parameters_serializer(pars)]
 
+
+    @staticmethod
+    def load_params(*input_values, **kwargs):
+        """Batch load of parameters from the `dcc.Input`
+        """
+        state_values = input_values[1]
+        return list(json.loads(state_values)[0].values())
```

Comment:
If I'm understanding this correctly, if `state_values` is an empty string this would fail right ? Especially since there is no validation for the value of `n_clicks` in the callback. AFAIK when the app is loaded `n_clicks` is set to `0` and so the callback is triggered when the app is loaded.